src/main/java/seedu/address/model/person/Person.java

```diff
@@ -30,7 +30,7 @@
      * Every field must be present and not null.
      */
     public Person(Name name, Phone phone, Email email, Mod mod, Group group, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, mod, group, tags);
+        requireAllNonNull(name, phone, email, mod, tags);
```

Comment:
From the previous implementation, group will never be null. It should be minimally a `new Group()`, so the requireAllNonNull should still include group here, i think